### ISSUE TITLE
Hide bookmark folder with password

### ISSUE THREAD

**Issue by macapple194** [xbrowsersync/app#206]
Include this feature on xBrowserSync (https://github.com/rharel/webext-private-bookmarks).
I think it may be a lot useful to have a folder that can be hidden if necessary, but included on sync service.

**Comment by albatros667** [comment]
it would be good, although looking for privacy I prefer to block the browser with password, for that use the extension LockPW

**Comment by macapple194** [comment]
Yes, but this way you protect the whole browser and you can't let anyone else browse. Instead with this feature you would hide the folders you are interested in even from those you allow to use your pc.
